kaggle_environments/core.py

```diff
@@ -121,9 +121,9 @@ def __init__(
             raise InvalidArgument("Renderer is not Callable.")
         self.renderer = renderer
 
-        if callable(html_renderer):
-            html_renderer = html_renderer()
-        self.html_renderer = get(html_renderer, str, "")
+        if not callable(html_renderer):
```

Comment:
Currently renderer returns a function Function(state, environment) which is called upon rendering. In case of html_renderer expectation is to return HTML code of the renderer, which makes it impossible to execute per-render Python logic. With this change interface of renderer and html_renderer is the same. In case of GFootball we use it by rendering video in Python and sending it to the client for viewing.